src/icloudpd/base.py

```diff
@@ -488,7 +488,7 @@ def report_version(ctx: click.Context, _param: click.Parameter, value: bool) ->
     is_flag=True,
 )
 @click.option(
-    "--keep-recent-days",
+    "--keep-icloud-recent-days",
     help="When using --delete-after-download, do not delete photos newer than this many days",
```

Comment:
We are using `-keep-icloud-recent-days` _instead_ of `--delete-after-download` according to RFC #1046. Also need to check that both params are never used together.

Reply:
Updated help. Added check below for the flags being used together.